Jenkinsfile

```diff
@@ -470,355 +453,395 @@ def testGo(prefix, packagesToTest) {
+        flags: '-race',
+        timeout: '30s',
+      ],
+      'github.com/keybase/client/go/kbfs/libkey': [
+        flags: '-race',
+        timeout: '5m',
+      ],
+      'github.com/keybase/client/go/kbfs/libkbfs': [
+        flags: '-race',
+        timeout: '5m',
+      ],
+      'github.com/keybase/client/go/kbfs/libpages': [
+        flags: '-race',
+        timeout: '30s',
+      ],
+      'github.com/keybase/client/go/kbfs/libpages/config': [
+        flags: '-race',
+        timeout: '30s',
+      ],
+      'github.com/keybase/client/go/kbfs/search': [
+        flags: '-race',
+        timeout: '30s',
+      ],
+      'github.com/keybase/client/go/kbfs/simplefs': [
+        flags: '-race',
+        timeout: '2m',
+      ],
+      'github.com/keybase/client/go/kbfs/test': [
+        name: 'kbfs_test_race',
+        flags: '-race',
+        timeout: '12m',
+      ],
+      'github.com/keybase/client/go/kbfs/tlf': [
+        flags: '-race',
+        timeout: '30s',
+      ],
+      'github.com/keybase/client/go/kbfs/tlfhandle': [
+        flags: '-race',
+        timeout: '30s',
+      ],
+    ],
+    test_windows_go_: [
+      '*': [],
+      'github.com/keybase/client/go/systests': [
+        disable: true,
+      ],
+      'github.com/keybase/client/go/chat': [
+        disable: true,
+      ],
+    ],
+  ]
+  def getOverallTimeout = { testSpec ->
+    def timeoutMatches = (testSpec.timeout =~ /(\d+)([ms])/)
+    return [
+      time: 1 + (timeoutMatches[0][1] as Integer),
+      unit: timeoutMatches[0][2] == 's' ? 'SECONDS' : 'MINUTES',
+    ]
+  }
+  def defaultPackageTestSpec = { pkg ->
+    def dirPath = pkg.replaceAll('github.com/keybase/client/go/', '')
+    def testName = dirPath.replaceAll('/', '_')
+    return [
+      name: testName,
+      flags: '',
+      timeout: '30m',
+      dirPath: dirPath,
+    ]
+  }
+  def getPackageTestSpec = { pkg ->
+    if (testSpecMap[prefix].containsKey(pkg)) {
+      if (testSpecMap[prefix][pkg]) {
+        def testSpec = testSpecMap[prefix][pkg]
+        if (testSpec['disable']) {
+          return false
         }
-        return defaultPackageTestSpec(pkg)
-      }
-      if (testSpecMap[prefix].containsKey('*')) {
-        return defaultPackageTestSpec(pkg)
+        return defaultPackageTestSpec(pkg) + testSpec
       }
-      return false
+      return defaultPackageTestSpec(pkg)
+    }
+    if (testSpecMap[prefix].containsKey('*')) {
+      return defaultPackageTestSpec(pkg)
+    }
+    return false
+  }
+  stage("Setup Test Suite") {
+  packagesToTest.each { pkg, _ ->
+    def testSpec = getPackageTestSpec(pkg)
+    if (!testSpec) {
+      return
     }
-    packagesToTest.each { pkg, _ ->
-      def testSpec = getPackageTestSpec(pkg)
-      if (!testSpec) {
-        return
-      }
-
-      println "Running go vet for ${pkg}"
-      sh "go vet ${pkg} || (ERR=\$? && echo \"go vet failed with error code \$ERR\" && exit \$ERR)"
```

Comment:
I don't see this being restored anywhere...

Reply:
golangci-lint does it